microcosm_flask/conventions/logging_level.py

```diff
@@ -69,28 +84,103 @@ class LoggerSchema(Schema):
     children = fields.List(fields.Nested("LoggerSchema"), required=True)
 
 
-class LoggingLevel(object):
-    pass
+class UpdateConditionalLoggingLevelSchema(Schema):
+    duration = fields.Float(required=False, missing=300.0)
```

Comment:
There's no reason not to use float since that's the time unit in Python; I chose it because it theoretically lets us write sleep-based tests with sub-second intervals, though I chose to just mock out time.